src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -295,10 +294,10 @@ private void setupKeyboardShortcuts() {
             if (UNDO_LABEL_CHANGES.match(event)) {
                 ui.triggerNotificationAction();
             }
-            for (KeyCodeCombination key : JUMP_TO_FIRST_ISSUE_KEYS) {
-                if (key.match(event)) {
+            for (Integer nKey:JUMP_TO_NTH_ISSUE_KEYS.keySet()) {
```

Comment:
Also you might want to iterate through the `entrySet` instead, since you need both key and value in the loop.